doc/source/install.rst

```diff
@@ -187,8 +187,7 @@ installed), make sure you have `pytest
     pandas version 0.18.0
     numpy version 1.10.2
```

Comment:
iI would update the pandas  & numpy version here as well (you can just change the text) to something more current

Reply:
actually this is from an older nose run, can you fix up to show what pytest prints.